artifacts/README.md

```diff
@@ -0,0 +1,127 @@
+# Kubernetes Artifacts
+
+This directory the manifest that are used to perform artifact promotion for
+the Kubernetes project.
```

Comment:
🤔  I'm not sure about the meaning of this (forgive my poor English). Maybe something like : 
```suggestion
This directory contains the manifests that are used to perform artifact promotion for
the Kubernetes project.
```

Reply:
@ameukam -- Nice catch! The wording was off and I've pulled in your suggestion. :)